### PR TITLE
Sync the SECURITY.md file with our HackerOne policy

### PR DESCRIPTION
**Pull request by desrosj**
Trac ticket: https://core.trac.wordpress.org/ticket/57937

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by hellofromtonya — APPROVED** [review]
Matches the HackerOne Policy content 👍

**Comment by peterwilsoncc** [comment]
Just so you're aware when committing these changes, `grunt patch` doesn't work with `SECURITY.md` so you'll need to do something like `patch -p0 < ${gh pr diff 4241 --repo=WordPress/wordpress-develop}` to get this in to your commit checkout. (Unless I've messed up and it's `-p1`.)

File: SECURITY.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Security Policy
 
-Full details of the WordPress Security Policy can be found on [HackerOne](https://hackerone.com/wordpress). You can also read more in a detailed white paper about [WordPress Security](https://wordpress.org/about/security/).
+WordPress is an open-source publishing platform. The WordPress Security Team believes in Responsible Disclosure by alerting the security team immediately and privately of any potential vulnerabilities.
+
+Our HackerOne program covers the Core software, as well as a variety of related projects and infrastructure.
+
+Full details of the WordPress Security Policy and the list of covered projects and infrastructure can be found on [HackerOne](https://hackerone.com/wordpress). You can also read more in a detailed white paper about [WordPress Security](https://wordpress.org/about/security/).
 
 ## Supported Versions
```
Comment (peterwilsoncc) [inline SECURITY.md:9]: It's outside the diff but let's remove the line that follows, it's from the template used to set up this file.

```diff
- Use this section to tell people about which versions of your project are
- currently being supported with security updates.

**Review by peterwilsoncc — APPROVED** [review]

**Comment by peterwilsoncc** [comment]
Committed in [r55670](https://core.trac.wordpress.org/changeset/55670) / https://github.com/WordPress/wordpress-develop/commit/70054b77afd75d17d5602d4cd445bdfb9e3ee62d

**Pull request closed without merging**